### ISSUE TITLE
UI: Click to Jump to Resource When Waiting on It

### ISSUE THREAD

**Issue by rwoll** [tilt-dev/tilt#5264]
## Describe the Feature You Want

In the Tilt UI, in the resource side bar, when a resource has a label of _Waiting on <other_resource>_ the _<other_resource>_ text should be a clickable link to the resource we are waiting on (so we can click and see it's logs).

If we're waiting on multiple resources and the text _Waiting on <num> resources" is shown, you can hover over that text and will see a list/tooltip of the links to the other resources.
 
### Current Behavior

The informational text appears, but it's not clickable/linked (or in the case of multiple resources, you're only left with a number).

### Why Do You Want This?

I see this message quite a bit, and my first response is to then go and search for the dep resource and click on the dependency to see what its logs are saying it's doing, so it would be nice to have a clickable shortcut with the message itself!

**Comment by nicks** [comment]
this is an interesting idea! i'm not totally convinced that this is the exact right UI, but point taken that it would be good to have a way to navigate between resources and the resources they're waiting on. cc @SurbhiSGupta

**Comment by rwoll** [comment]
Agreed! I quickly prototyped this today for the singleton case, and while I like the idea, once I clicked through to the dependent resource, I was then stuck there when it finished and had the reverse problem: how do I quickly jump back to the now updating resource that was previously waiting.

Im sure there's some enhancement here, but I would not implement my suggestion verbatim 😀

**Comment by rwoll** [comment]
I like the "Alerts on Top" quick filter, perhaps other quick filters could be useful ("Updating", "Waiting", etc.)

**Comment by rwoll** [comment]
Another filter that might be useful is "Initialized/Activated".

Our non-core services start with auto_init=False, and the few services you init/trigger to work on temporarily get lost in the side bar with the rest.
